### DISCUSSION TITLE
import path too long,how to make it like yarn?

### DISCUSSION THREAD

**Discussion by pincman** [pnpm/pnpm#3607] [Q&A]
only pnpm 
![image](https://user-images.githubusercontent.com/74093067/126402626-f125abb3-4a5f-4969-ab5b-f3c03a4ad919.png)
yarn or pnpm+`"preserveSymlinks": true` in tsconfig
![image](https://user-images.githubusercontent.com/74093067/126402734-b334c651-d985-44d5-bc2b-27c6bb36054b.png)
but `"preserveSymlinks": true` build error by `vitejs`,so in my case `preserveSymlinks` must be `false`
work version of typescript is 4.4.0-dev

**Comment by zkochan** [comment]
This seems like an issue with the code editor that you use.

**Reply by pincman** [reply]
I use vscode,is there any solution for vscode? thx

**Reply by zkochan** [reply]
Probably this https://github.com/microsoft/TypeScript/issues/44511

**Reply by pincman** [reply]
this does not solve the issue,i have installed typescript@next,and used it in vscode,but the problem still exists
![image](https://user-images.githubusercontent.com/74093067/126453480-6c434b4c-824c-4489-8fa6-83e77cf9ac5a.png)
![image](https://user-images.githubusercontent.com/74093067/126453508-7aee38e5-c6f7-4733-a080-28a4a4ac8325.png)
